types.mjs

```diff
@@ -0,0 +1,23 @@
+import fs from 'fs';
+
+// Create a regex that matches any string starting with Class< and ending with >
+const regex = /Class<(.*?)>/g;
+const paths = [
+    './types/framework/components/script/component.d.ts',
+    './types/script/script-attributes.d.ts',
+    './types/script/script-registry.d.ts',
+    './types/script/script.d.ts'
+];
+
+paths.forEach(path => {
+    let dts = (fs.readFileSync(path, 'utf8')).toString();
```

Comment:
Is the `(...).toString()` here and below really needed? I've never seen it returning a `Buffer` when specifying `utf8`

Reply:
Ha, very cool. I love it when you point these things out to me! Fixed.